.github/workflows/build-push.yaml

```diff
@@ -0,0 +1,65 @@
+name: "Build & Push"
+
+on:
+  push:
+    branches:
+      - "develop"
+      - "main"
+    paths:
+      - "packages/**"
+
+jobs:
+  build_push_image:
+    name: ${{ matrix.image_name }}
+    runs-on: ubuntu-latest
+    environment: dev
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image_name: probing-api
+            dockerfile_path: packages/probing-api
+          - image_name: probing-caller
+            dockerfile_path: packages/probing-caller
+          - image_name: probing-eservice-operations
+            dockerfile_path: packages/probing-eservice-operations
+          - image_name: probing-eservice-registry-reader
+            dockerfile_path: packages/probing-eservice-registry-reader
+          - image_name: probing-eservice-registry-updater
+            dockerfile_path: packages/probing-eservice-registry-updater
+          - image_name: probing-response-updater
+            dockerfile_path: packages/probing-response-updater
+          - image_name: probing-scheduler
+            dockerfile_path: packages/probing-scheduler
+          - image_name: probing-statistics-api
+            dockerfile_path: packages/probing-statistics-api
+          - image_name: probing-telemetry-writer
+            dockerfile_path: packages/probing-telemetry-writer
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: normalize branch name for tagging
+        run: |
+          NORM_TAG=$(echo "$GITHUB_REF_NAME" | tr -s "/" "-")
+          echo "NORM_TAG=$NORM_TAG" >> $GITHUB_ENV
```

Comment:
```suggestion
```
This was needed when the pipeline triggered on `feature/...` branches. 
Now it triggers only on `main`, `develop` so the normalization is not needed.